reconciler/reconciler.go

```diff
@@ -111,6 +117,10 @@ func (r *reconciler) reconcile() {
 			}
 		}
 
+		if match == nil || !proto.Equal(desiredService.HealthCheck, match.HealthCheck) {
+			r.checker.SetHealthCheck(desiredService.Id, desiredService.HealthCheck)
```

Comment:
Is it worth logging that a healthcheck is being set, similar to the above `Adding virtual service: %v` message?

Reply:
I've added it to the health checker.